### ISSUE TITLE
Can't build project - Typescript problem

### ISSUE THREAD

**Issue by vprado-dev** [zackradisic/node-soundcloud-downloader#122]
I added this library in my discord-bot typescript project, but i'm getting so many errors when I'm trying to build it.

Do you know something about it? I think that's some problems with types, but I don't know how to solve it

![image](https://user-images.githubusercontent.com/39499949/140181166-fff59a55-d424-4183-8960-e49a9309b83d.png)

**Comment by North-West-Wind** [comment]
Add `"noImplicitAny": false` to `compileOptions` in `tsconfig.json`

**Comment by ErezCsillag** [comment]
> Add `"noImplicitAny": false` to `compileOptions` in `tsconfig.json`

Maybe do not use any?
it's a library it should be "plug and play", which it is totally not.
it makes the library unusable for me

**Comment by vprado-dev** [comment]
> 

I totally agree with you, I don't have to change my entire project typescript config to use a single library

**Comment by vprado-dev** [comment]
> > Add `"noImplicitAny": false` to `compileOptions` in `tsconfig.json`
> 
> Maybe do not use any? it's a library it should be "plug and play", which it is totally not. it makes the library unusable for me

I totally agree with you, I don't have to change my entire project typescript config to use a single library
